src/main/java/com/cloudbees/jenkins/plugins/bitbucket/FillErrorResponse.java

```diff
@@ -0,0 +1,35 @@
+package com.cloudbees.jenkins.plugins.bitbucket;
+
+import hudson.Util;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletResponse;
+import jenkins.model.Jenkins;
+import org.kohsuke.stapler.HttpResponse;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+
+// TODO replace with corresponding core functionality once Jenkins core has JENKINS-42443
+class FillErrorResponse extends IOException implements HttpResponse {
+
+    private final String message;
+    private final boolean clearList;
+
+    public FillErrorResponse(String message, boolean clearList) {
+        this.message = message;
```

Comment:
It's not really used as an actual exception, do you feel strongly about using it as such?